README.md

```diff
@@ -138,12 +145,109 @@ make test-ts-binary-rococo
 
 Remember to run the clean-up afterwards.
 
-## extend the leasing period
+## License
 
-The default leasing duration for parachain is 1 day, in case you want to extend it (even after it's downgraded to parathread), simply do a `forceLease` via sudo, it should be upgraded to parachain soon again and start to produce blocks.
+GPLv3
 
-![image](https://user-images.githubusercontent.com/7630809/135689832-1f57cd5c-7f83-4fce-9bb0-832b77a38dcc.png)
 
-## License
+## How to Build and Run Parachain and Tee-worker
 
-GPLv3
+## Preparation
+
+1. env: [Setup **SGX TEE** Environment](https://web3builders.notion.site/Setup-SGX-TEE-Environment-68066770831b45b7b632e682cf159477?pvs=4) 
+2. add VSCode configs for indexing and parsing tee-worker codes
+
+```
+"rust-analyzer.linkedProjects": [
+        "tee-worker/Cargo.toml"
+    ],
+```
+
+## Build
+
+We need to build the `integritee-service` before we launch the entire system
+
+```
+git clone https://github.com/litentry/litentry-parachain
+cd ./litentry-parachain/tee-worker
+source /opt/intel/sgxsdk/environment
+SGX_MODE=SW make
+```
+
+## Launch
+
+### 1. Start a local docker setup
+
+In order to create a local docker setup, you can run the following command
+```
+./local-setup/launch.py local-setup/github-action-config-one-worker.json
+```
+This will create three docker containers, 2 Relay Chain Validators and 1 Parachain Collator. However it will be using the default ports as present in .env.dev . If you want to run the system by offseting the default ports by a certain value. You can run this command instead:
+
+```
+./local-setup/launch.py local-setup/development-worker.json local-docker 100
```

Comment:
Please use same config file as example.

Reply:
This has me confused, can you kindly explain what mean better?

Reply:
there are some example commands you listed inside this README file. Please try to use same example file, which is inside repo.

Reply:
@BillyWooo I'm confused too tbh; do you mean the `local-setup/development-worker.json` file? That's [in the repo](https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/local-setup/development-worker.json), and it's the one the developer docs on Notion point to (/cc @felixfaisal). If this file is not the right one, which one should be referenced exactly?

Reply:
btw I tried this on dev just now and it errored out; I think the command line args changed now to include the temp directory? /cc @felixfaisal

Reply:
> I'm confused too tbh; do you mean the local-setup/development-worker.json file? That's [in the repo](https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/local-setup/development-worker.json), and it's the one the developer docs on Notion point to (/cc @felixfaisal).

Yeah so we can perform offset with `development-worker.json`, since doing offset was only meant for development setups especially cases wherein multiple share the same system. We use `github-action-one-worker.json` when we are using default ports for `Parachain` , `Relaychain` and `Worker`, it will just fail if any of these ports are occupied by another process

Reply:
> btw I tried this on dev just now and it errored out; I think the command line args changed now to include the temp directory? /cc @felixfaisal

I think I replied to this on Slack to @cryptoade1, There's two changes one is including `tmp` directory, it's optional again this is meant to be used for scenarios wherein multiple people share the system and we have to use different `tmp` directory and the hardcoded directories will have locks for the other. The other change is defining the scope of the `log` via a `toml` file